cafebabel/static/js/script.js

```diff
@@ -56,3 +56,56 @@ Array.from(document.querySelectorAll('.authentication-form > div > input')).forE
         parent.classList.remove('active', 'completed')
     })
 })
+
+/* highlight file upload area on hover or dragenter */
+function addListenerMulti(element, events, fn) {
+    events.split(' ').forEach(event => element.addEventListener(event, fn, false)) /* https://stackoverflow.com/questions/8796988/binding-multiple-events-to-a-listener-without-jquery */
+}
+const fileInput = document.querySelector('.file input')
+const dropArea = document.querySelector('canvas')
+if (fileInput) {
+    addListenerMulti(fileInput, 'dragenter focus click', () => {
+        dropArea.classList.add('active')
+    })
+    addListenerMulti(fileInput, 'dragleave blur drop', () => {
+        dropArea.classList.remove('active')
+    })
+    fileInput.addEventListener('change', () => {
+        document.querySelector('.file label').innerHTML(' ')
+    })
+}
+
+/* tv noise effect https://codepen.io/eugene-bulkin/pen/zEgyH how to know how "heavy" is it? */
+const canvas = document.querySelector('canvas')
+if (canvas) {
+    const context = canvas.getContext('2d')
+
+    function resize() {
+        canvas.width = window.innerWidth
+        canvas.height = window.innerHeight
+    }
+    resize();
+    window.onresize = resize
+
+    function noise(context) {
```

Comment:
Shouldn't we move this function into its own file? (eg: _noise.js_)

Reply:
yep, cause every time the window size is change we call this function ?